index.html

```diff
@@ -35,12 +35,12 @@
   </div>
     <header>
       <div class="toolbar">
+        <!-- Nav Mobile - Desktop -->
         <div class="mobileNav">
           <nav>
             <button type="button" class="hamburguer">&#9776;</button>
           </nav>
         </div>
-        <!-- Nav Mobile - Desktop -->
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you add an interaction on the Main Program cards when a cards is hovered
![Screenshot from 2023-02-18 23-20-13](https://user-images.githubusercontent.com/83042742/219900073-7f0a0dd9-aa93-487c-893d-906760c8dc88.png)

Reply:
> Correction

Add optional comment

![imagen](https://user-images.githubusercontent.com/90431831/219906199-90a1d96b-9551-43bc-91f4-4f4ee7149971.png)